admin-data-explorer-web/src/main/scala/net/wiringbits/webapp/utils/ui/web/components/EditGuesser.scala

```diff
@@ -1,27 +1,41 @@
 package net.wiringbits.webapp.utils.ui.web.components
 
-import japgolly.scalajs.react.vdom.html_<^.VdomNode
+import japgolly.scalajs.react.vdom.html_<^._
 import net.wiringbits.webapp.utils.api.models.AdminGetTables
 import net.wiringbits.webapp.utils.ui.web.facades.reactadmin._
-import net.wiringbits.webapp.utils.ui.web.models.ColumnType
+import net.wiringbits.webapp.utils.ui.web.models.{ColumnType, TableAction}
 import net.wiringbits.webapp.utils.ui.web.utils.ResponseGuesser
+import org.scalajs.dom
 
 object EditGuesser {
 
-  def apply(response: AdminGetTables.Response.DatabaseTable) = {
+  def apply(response: AdminGetTables.Response.DatabaseTable, tableActionsMaybe: Option[TableAction]) = {
     val fields = ResponseGuesser.getTypesFromResponse(response)
-    val inputs: List[VdomNode] = fields.map { fieldType =>
-      fieldType.`type` match {
-        case ColumnType.Date => DateTimeInput(_.source := fieldType.name, _.disabled := fieldType.disabled)
-        case ColumnType.Text => TextInput(_.source := fieldType.name, _.disabled := fieldType.disabled)
-        case ColumnType.Email => TextInput(_.source := fieldType.name, _.disabled := fieldType.disabled)
-        case ColumnType.Reference(reference, source) =>
-          ReferenceInput(_.source := fieldType.name, _.reference := reference)(
-            SelectInput(_.optionText := source, _.disabled := fieldType.disabled)
-          )
+    val inputs: List[VdomNode] = fields
+      .map { fieldType =>
+        fieldType.`type` match {
+          case ColumnType.Date => DateTimeInput(_.source := fieldType.name, _.disabled := fieldType.disabled)
+          case ColumnType.Text => TextInput(_.source := fieldType.name, _.disabled := fieldType.disabled)
+          case ColumnType.Email => TextInput(_.source := fieldType.name, _.disabled := fieldType.disabled)
+          case ColumnType.Reference(reference, source) =>
+            ReferenceInput(_.source := fieldType.name, _.reference := reference)(
+              SelectInput(_.optionText := source, _.disabled := fieldType.disabled)
+            )
+        }
       }
-    }
-    Edit()(
+
+    val buttons = tableActionsMaybe
+      .map { tableActions =>
```

Comment:
Actually, it would be ideal to not assume that the action is a button, some actions could need an input, still, this is fine for now, just update this to allow many actions